tests/conformance/bindings/bindings.go

```diff
@@ -143,23 +143,27 @@ func ConformanceTests(t *testing.T, props map[string]string, inputBinding bindin
 	})
 
 	t.Run("ping", func(t *testing.T) {
-		errInp := bindings.PingInpBinding(inputBinding)
-		// TODO: Ideally, all stable components should implenment ping function,
-		// so will only assert assert.Nil(t, err) finally, i.e. when current implementation
-		// implements ping in existing stable components
-		if errInp != nil {
-			assert.EqualError(t, errInp, "ping is not implemented by this input binding")
-		} else {
-			assert.Nil(t, errInp)
+		if config.HasOperation("read") {
+			errInp := bindings.PingInpBinding(inputBinding)
+			// TODO: Ideally, all stable components should implenment ping function,
+			// so will only assert assert.Nil(t, err) finally, i.e. when current implementation
+			// implements ping in existing stable components
+			if errInp != nil {
+				assert.EqualError(t, errInp, "ping is not implemented by this input binding")
+			} else {
+				assert.Nil(t, errInp)
+			}
 		}
-		errOut := bindings.PingOutBinding(outputBinding)
-		// TODO: Ideally, all stable components should implenment ping function,
-		// so will only assert assert.Nil(t, err) finally, i.e. when current implementation
-		// implements ping in existing stable components
-		if errOut != nil {
-			assert.EqualError(t, errOut, "ping is not implemented by this output binding")
-		} else {
-			assert.Nil(t, errOut)
+		if config.HasOperation("operations") {
```

Comment:
Is this a new operation? If so, please make sure you document it at the  top of [`tests/config/bindings/tests.yml`](https://github.com/dapr/components-contrib/blob/a4b27ae49b7c99820c6e921d3891f03334692714/tests/config/bindings/tests.yml#L1-L2)

Reply:
Please just document it in the bindings/test.yaml file then (and make sure that it's added to all test cases where it matters)

Reply:
Sorry @ItalyPaleAle, I’m not following you,

I did not create anything new, it’s just a matter of reusing of what existe and applying consistently with the initialization tests.

previously, the ping tests are executed regardless wether init was invoked or not. The operations that I’m using for this are the same for the lines above on the same file.

what do you mean for documenting those?

Reply:
You’re running that test if the operation `operations` is specified in the config/bindings/test.yml file. Unless you explicitly add that operation in the YAML, the branch with the test you added will not be executed.

Reply:
Do you know that the operation `operations` is not new, right?

If there are tests that had been executed because of `operations` is specified so they will continue to run.

And I don’t need to add “operations” for the tests that don’t declare anything because I’m fixing exactly the case where none operations are specified but ping tests were executed

Unless you meant to create a new operation “ping”, I guess we are fine.

Please, if you still thinking that something should be done, would you mind to point exactly where and what?

Reply:
> Do you know that the operation operations is not new, right?

No, I wasn't clear on that. I didn't see it in the YAML file documented so that's why I was asking "Is this a new operation". Thanks for confirming it's not a new one.